src/main/java/racingcar/model/RacingCar.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.model;
+
+import java.util.Random;
+
+public class RacingCar {
+    private static final String ERR_NAME_LENGTH = "Names longer than 5 characters are not allowed.";
+    private static final int NAME_MAX_LENGTH = 5;
+    private static final int THRESHOLD_NUMBER = 4;
+    private static final int START_POSITION = 1;
+    private static final int STEP = 1;
+
+    private int position;
+    private String name;
+
+    public RacingCar(String name) {
+        // validation
+        validateCarName(name);
+
+        // setting
+        this.name = name;
+        this.position = START_POSITION;
+    }
+
+    private void validateCarName(String name) {
+        if (name.length() > NAME_MAX_LENGTH) {
+            throw new RuntimeException(ERR_NAME_LENGTH);
```

Comment:
RuntimeException은 상위 예외 클래스인데요
조금 더 구체적인 표준 예외를 사용하면 어떨까요?
```suggestion
            throw new IllegalArgumentException(ERR_NAME_LENGTH);
```